main.py

```diff
@@ -116,6 +127,8 @@
     model = NeRFNetwork(opt)
 
     print(model)
+    for i in model.encoder.parameters():
+        print(i.shape)
```

Comment:
I guess this is for debugging can be removed?

Reply:
Thanks for the catch. Have removed.